internal/vds/vds_test.go

```diff
@@ -802,6 +804,38 @@ func TestHorizon(t *testing.T) {
 	}
 }
 
+func TestHorizonSurface(t *testing.T) {
```

Comment:
Maybe `TestHorizonUnalignedWithSeismic` or something to that effect. `HorizonSurface` doesn't communicate _what_ you are hoping to test.